packages/lib/src/Button.wc.svelte

```diff
@@ -12,9 +14,10 @@
   <a {href} class="button" class:outline {onclick}><slot /></a>
 {:else}
   <button
-    class="button bg-yellow-500"
+    class="button bg-yellow-500 {size}"
     class:outline
     class:disabled
+    {part}
```

Comment:
Wonder if this will work? Guess so right?

Reply:
Yes. {varName} is a shorthand in svelte that means varName={varName}

If varName = undefined it will be ignored in the compiled output